### ISSUE TITLE
HTML file with Ethers.js

### ISSUE THREAD

**Issue by Adam-Koza** [ricmoo/meeseeks-app#8]
Hey Richard, 

Cool project. I was trying use it to render a html page that uses ethers.js to read a smart contract on Ropsten through metamask, but it's unable to connect. 

File works fine if I host it on my local machine with a simple node server. 

https://0xgbno6fjji3zmi4whsthul5wcjslaixpbjle3qycq3wl4xnrbi54ww.meeseeks.app

IPFS Multihash: QmaaazCRJBzBb8ihDuTcLx7Ka6bFdKEPE8BM5cQRxvE3do

What is preventing this from working?

Thanks!

**Comment by ricmoo** [comment]
What do you get? I get this:

![f86c12a6-03b0-4542-92f6-5a9c0bf3e21e](https://user-images.githubusercontent.com/975687/51721798-3d8a6380-2021-11e9-9d8b-4ed407e6ca0b.png)

**Comment by Adam-Koza** [comment]
Yeah that is what I get, but it should be populated by commits based on data read from the smart contract.

**Comment by ricmoo** [comment]
Hmmm. I’ll have to take a look later on a computer. I’m afk right now. :s

**Comment by Adam-Koza** [comment]
No problem, thanks for the prompt response! 

This is what it should look like when properly connected:

![screenshot from 2019-01-24 21-49-11](https://user-images.githubusercontent.com/16749083/51722042-108a8080-2022-11e9-8c24-15d10561ab59.png)
